lib/locales/el.ts

```diff
@@ -0,0 +1,501 @@
+import { Locale } from '../types/locale';
+
+export const locale: Locale = {
+  id: 'el',
+
+  seasons: {
+    advent: {
+      season: 'Περίοδος της Παρουσίας',
+      weekday: '$t(weekdays:{{dow}}) της $t(ordinals:{{week}}, { "context": "abbreviation" }) Εβδομάδας της Παρουσίας',
+      sunday: '$t(ordinals:{{week}}) Κυριακή της Παρουσίας',
+      privileged_weekday: '{{day}} $t(months:11)',
+    },
+
+    christmas_time: {
+      season: 'Περίοδος των Χριστουγέννων',
+      day: '$t(weekdays:{{dow}}) των Χριστουγέννων',
+      octave: '{{count}}η ημέρα της Ογδοάδας των Χριστουγέννων',
+      second_sunday_after_christmas: 'Δεύτερη Κυριακή μετά τα Χριστούγεννα',
+      before_epiphany: '{{day}} $t(months:0)',
+      after_epiphany: '$t(weekdays:{{dow}}) μετά την πανήγυρη των Θεοφανίων',
+    },
+
+    ordinary_time: {
+      season: 'Kοινή περίοδος του έτους',
```

Comment:
I believe you used Latin `K` instead of Greek `Κ`.

```suggestion
      season: 'Κοινή περίοδος του έτους',
```